pallets/auctions/src/benchmarking.rs

```diff
@@ -104,7 +104,7 @@ mod tests {
 		new_test_ext().execute_with(|| {
 			assert_ok!(test_benchmark_create_auction::<Test>());
 			assert_ok!(test_benchmark_bid_value::<Test>());
-			assert_ok!(test_benchmark_delete_auction::<Test>());
+			assert_ok!(test_benchmark_destroy_auction::<Test>());
```

Comment:
btw, for better quality of life - you can use 

`use frame_benchmarking::impl_benchmark_test_suite;`

using this, you dont have to explicty add each benchmarked method as test. It will auto implement/add them.

Example of usage [here](https://github.com/galacticcouncil/Basilisk-node/pull/235/commits/ca4147c2520f94b33f3ab6b63d2e566e737c2906)